### PR TITLE
check if an user is present in the channel for nickchange

### PR DESCRIPTION
**Pull request by Junky2k**
i only check if the channel has been tracked, not if the user was present in the channel.

i fu**ed up, i'm sorry :D

### CODE REVIEW

File: lib/bot.js [hunk 1]
```diff
@@ -129,9 +129,11 @@ class Bot {
...
-          this.channelUsers[channel].delete(oldNick);
-          this.channelUsers[channel].add(newNick);
-          this.sendExactToDiscord(channel, `*${oldNick}* is now known as ${newNick}`);
+          if (this.channelUsers[channel].has(oldNick)) {
+            this.channelUsers[channel].delete(oldNick);
+            this.channelUsers[channel].add(newNick);
+            this.sendExactToDiscord(channel, `*${oldNick}* is now known as ${newNick}`);
+          }
         } else {
           logger.warn(`No channelUsers found for ${channel} when ${oldNick} changed.`);
```
Comment (ekmartin) [inline lib/bot.js]: To be honest I don't think we really need the logger warning here, so maybe just fold the two `if`s into one and do `if (this.channelUsers[channel] && this.channelUsers[channel].has(oldNick)) {`?

Comment (ekmartin) [inline lib/bot.js]: Or what do you think @Throne3d?

**Comment by ekmartin** [comment]
no worries, thanks for the fix :)

File: lib/bot.js (hunk 1 above)
Comment (Throne3d) [inline lib/bot.js]: The logger warning probably isn't necessary – it was mostly to get around the issue with lowercasing channel names, in the other events.

Comment (ekmartin) [inline lib/bot.js]: Eh, I guess not. Just a bit cleaner code, but it'll do.

**Pull request merged**